pkg/testdata/error-test-shard-identity.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
What's the error in this file?

Reply:
In this one "metadata": "test-value" is set as string but expected with IdentityMetdata type